__tests__/app.test.js

```diff
@@ -39,4 +39,35 @@ describe("GET /api/categories", () => {
   });
 });
 
-
+describe.only("GET /api/reviews", () => {
+  test("200: retrieves list of reviews from database", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toHaveLength(13);
+      });
+  });
+  test("200: retrieved reviews contain a comment_count representing the number of comments that reference that review", () => {
```

Comment:
I like that you've separated this logic out into another test, I think its a little brittle in terms of the test structure but its doing the job 👍